assets/i18n/ko.po

```diff
@@ -5740,7 +5742,7 @@ msgstr ""
 
 #, javascript-format
 msgid "Return, but ${ doNotRemove } ${ bless } / ${ curse } tokens"
-msgstr "${ bless } / ${ curse } 토큰 수는 ‘토큰 되돌리기’ 후 자동 반영됨 ${ doNotRemove }"
+msgstr "‘토큰 되돌리기’하면 자동 반영: ${ bless } / ${ curse } 토큰을 따로 ${ doNotRemove }!"
```

Comment:
Sorry, I just adjusted it because I wanted the text to show up all inside the field. So I want to revert all related changes and only adjust the 'ko' entries.
... I'm not sure if my intentions came through to you.

Reply:
Yeah, so the app needs to keep all the placeholders.  But in this case I put in an alternative for KO that uses your suggested string here.

Reply:
If its better with your correction I don't mind maintaining two versions (this one will just not be used for Korean)?  LMK which way you prefer.

Reply:
I wanted to show `${doNotRemove}` entries without referencing them elsewhere.
- LIKE: `msgid: Return, but ${ doNotRemove } ${ bless } / ${ curse } tokens` -> `msgid: Return, but do not remove ${ bless } / ${ curse } tokens`

But I don't think it's necessary enough to have a separate entry.
Thank you for your kind response.